nginx.conf

```diff
@@ -9,6 +9,10 @@ server {
     }
 
     location /v1 {
```

Comment:
Do we care to expose any [Pelias endpoints](https://github.com/pelias/documentation/#endpoint-descriptions) besides `/v1/autocomplete`? If not, we can restrict this even further.